src/SonarQube.TeamBuild.Integration.Legacy/SonarQube.TeamBuild.Integration.Legacy.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net46</TargetFramework>
+    <AssemblyName>TeamBuild.SonarQube.Integration.Legacy</AssemblyName>
+    <RootNamespace>SonarQube.TeamBuild.Integration.Legacy</RootNamespace>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\SonarQube.Common\SonarQube.Common.csproj" />
+    <ProjectReference Include="..\SonarQube.TeamBuild.Integration\SonarQube.TeamBuild.Integration.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNet.WebApi.Client" version="5.2.2" />
+    <PackageReference Include="Microsoft.AspNet.WebApi.Core" version="5.2.2" />
+    <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" version="3.13.5" />
```

Comment:
Presumably all of the TFS specific references are no longer required in _SonarQube.TeamBuild.Integration.csproj_?

Reply:
Not sure, I'll test.